lib/mayaUsd/fileio/jobs/readJob.cpp

```diff
@@ -56,7 +56,7 @@
 #include <maya/MStatus.h>
 #include <maya/MTime.h>
 
-#include <ghc/filesystem.hpp>
+#include <boost/filesystem.hpp>
```

Comment:
@dj-mcg Could you clarify why you needed to make this change here? In fact, This PR should not compile at all on any platforms since we no longer support boost::filesystem.

Reply:
This was an oversight on my part. We change these internally since we still use boost, so this didn't show up in the diff I was looking at. I've corrected it